index.js

```diff
@@ -76,13 +76,17 @@ var InfluxDB = function (options) {
 }
 
 InfluxDB.prototype._parseResults = function (response, callback) {
+  var self = this
   var results = []
   _.each(response, function (result) {
     var tmp = []
     if (result.series) {
       _.each(result.series, function (series) {
         var rows = _.map(series.values, function (values) {
-          return _.extend(_.zipObject(series.columns, values), series.tags)
+          var columnsDesanitized = _.map(series.columns, self._desanitizeString)
+          var valuesDesanitized = _.map(values, self._desanitizeString)
+          var tagsDesanitized = _.forOwn(series.tags, self._desanitizeObjectIter.bind(self))
```

Comment:
This is via the Influx CLI, the results are from [`/query`](https://docs.influxdata.com/influxdb/v1.0/guides/querying_data/) API. This output format can be toggled like:

```
➜  ~ influx
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version v1.0.0
InfluxDB shell version: v1.0.0
> format json
> pretty
Pretty print enabled
> // all query outputs will be json now
```